app/controllers/api/databases.php

```diff
@@ -1574,7 +1574,7 @@ function createAttribute(string $databaseId, string $collectionId, Document $att
             Query::equal('databaseInternalId', [$db->getInternalId()])
         ], 61);
 
-        $limit = 64 - MariaDB::getNumberOfDefaultIndexes();
+        $limit = 64 - MariaDB::getCountOfDefaultIndexes();
```

Comment:
We should also expose this method from the `Database` class instead of accessing it like this. This couples us with MariaDB. Could also be addressed in a separate PR if requires another DB release.

Reply:
Separate PR